easy_my_coop_website_portal/views/easy_my_coop_website_portal_templates.xml

```diff
@@ -146,7 +146,9 @@
 
                 <p t-if="coop.gender">
                     <label>Gender:</label>
-                    <t t-esc="coop.gender" />
+                    <t
+                        t-esc="dict(coop._fields['gender']._description_selection(env))[coop.gender]"
```

Comment:
@carmenbianca  I'm surprised by this solution. It seems overcomplicated to me. Do you have an understanding of why this field is not translated while others are ?

The selection values (defined [here](https://github.com/coopiteasy/vertical-cooperative/blob/b6f35f5b6f1a99e1285555a80c35876a181aa120/easy_my_coop/models/partner.py#L142-L146)) should be automatically translated by odoo.

While you're at it, could you resolve the todo `# todo use oca partner_contact_gender` ?

Reply:
@robinkeunen I am very surprised also, but:

> The selection values (defined here) should be automatically translated by odoo.

This is not what happened. When doing `coop.gender` given `("male", "Male")`, the program printed `male`, not `Male`.

I searched quite a bit on how to get the selection _value_ given a selection, and the above kept popping up. There has to be a better way, but I don't know it.

Will look at the todo.